lib/plugins/bower-manifest/index.js

```diff
@@ -27,7 +27,9 @@ function linkFile(blob, key, value) {
 export default class BowerManifest {
 
   getPattern() {
-    return ['bower.json'];
+    return {
+      pathSubstrings: ['bower.json'],
+    };
```

Comment:
Mhhh I think there is no case where a plugin needs to return a GitHub-Class right? Therefore I would like to keep the plugin interface like it was before and do the GitHub-Class lookup behind the scenes in the `plugin-manager`.